packages/sitecore-jss/src/personalize/layout-personalizer.ts

```diff
@@ -56,7 +56,7 @@ export function personalizeComponent(
   if (variant === undefined && component.componentName === undefined) {
```

Comment:
Just curious... is this an expected case or just a fail-safe?

Reply:
As I understood, it's expected, can be in case if component is hidden, but doesn't have variant